src/config/lang/english.go

```diff
@@ -513,3 +513,8 @@ var (
 	ErrInitNotFound      = errors.New("this command requires a zarf-init package, but one was not found on the local system. Re-run the last command again without '--confirm' to download the package")
 	ErrUnableToCheckArch = errors.New("unable to get the configured cluster's architecture")
 )
+
+// Collection of reusable warn messages.
+var (
+	WarnSGetDeprecation = "Using sget to download resources is being deprecated and will removed in the v0.29.0 release of Zarf. Please use {@JPERRY} instead."
```

Comment:
```suggestion
	WarnSGetDeprecation = "Using sget to download resources is being deprecated and will be removed in the v0.31.0 release of Zarf. Please use https or oci where applicable instead."
```
We should give folks at least 2 versions for this I think though it would be worth discussing internally if we want to give more - `sget` is already gone from cosign but we may want to give more time (potentially).